VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -171,19 +172,9 @@
                         tools:targetApi="17" />
 
                     <Switch
+                        style="@style/edX.Widget.Switch"
```

Comment:
`android:switchStyle` works for me on Nexus 4 Jelly Bean Genymotion emulator, but your experience is further warning against relying on AOSP implementation I guess.